app/Application.php

```diff
@@ -80,8 +107,14 @@ public static function autocomplete()
         return $list;
     }
 
+    /**
+     * @param $appid
+     * @return mixed|null
+     */
     public static function getApp($appid)
     {
+        Log::debug("Get app triggered for: $appid");
```

Comment:
@KodeStar This should only be logging when in the .env file APP_DEBUG is set to true